lm_eval/evaluator.py

```diff
@@ -171,13 +185,20 @@ def evaluate(
         for req in reqs:
             cloned_reqs.extend([req] * req.repeats)
 
+        if (lm.world_size > 1) and (numpad > 0):
```

Comment:
@fattorib Seems this block is not indented enough, because `req` is defined inside loop, which finished working (above), and at this point `req` is one last of reqs, not each of reqs. Maybe this is intended behaviour, though.